test/__snapshots__/PublishCommand.js.snap

```diff
@@ -1,13 +1,6 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`[independent --canary --npm-tag=next --yes --exact] npm dist-tag add 1`] = `
```

Comment:
Likewise with the other snapshots that have turned into empty objects, those aren't very useful and should be replaced by `publishedTagInDirectories()` where it makes sense.